packages/content/src/lib/markdown.component.ts

```diff
@@ -64,12 +75,20 @@ export default class AnalogMarkdownComponent
   }
 
   updateContent() {
-    this.content$ = this.route.data.pipe(
-      map<Data, string>((data) => this.content ?? data['_analogContent']),
-      mergeMap((contentString) => this.renderContent(contentString)),
-      map((content) => this.sanitizer.bypassSecurityTrustHtml(content)),
-      catchError((e) => of(`There was an error ${e}`))
-    );
+    if (this.content && typeof this.content !== 'string') {
+      if (this.container) {
+        this.container.clear();
+        this.container.createComponent(this.content as any);
+        this.cdr.detectChanges();
```

Comment:
nit/suggestion: should probably grab the `ComponentRef` from `createComponent` and call `detectChanges()` on it instead.